robolectric-resources/src/main/java/org/robolectric/res/Attribute.java

```diff
@@ -85,7 +39,7 @@ public boolean isResourceReference() {
   }
 
   public boolean isStyleReference() {
-    return value.startsWith("?");
+    return isStyleReference(value);
```

Comment:
Ya, probably, but I'm planning on cutting a release after this is submitted and then in a follow up PR deleting Attribute + RobolAttributeSet, so it will be @Deprecated for one release.